Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM elixir:1.4.2
+
+ENV NODE_VERSION=7
+
+# Install system dependencies
+RUN apt-get -y update \
+    && apt-get -y install apt-transport-https build-essential curl git make locales locales-all inotify-tools
+
+# Install nodejs
+RUN curl -sL https://deb.nodesource.com/setup_$NODE_VERSION.x | bash - \
+    && apt-get -y install nodejs
+
+# Clean up apt temporary apt artefacts
+RUN apt-get clean -y \
+    && rm -rf /var/cache/apt/*
```

Comment:
Cool! Thanks!
And do you think it makes sense to join all the apt commands in favor of the container size? Or just remove this cleaning command and leave it as it is?